arm-web/2015-08-01-preview/swagger/logicManagementClient.json

```diff
@@ -0,0 +1,1299 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LogicManagementClient",
```

Comment:
Unfortunately the architecture is funny for these APIs. The API endpoints are hosted by logicApps RP but they belong to the Microsoft.Web namespace. That is why they are part of the WebsiteManagementClient composite client.